app/com/linkedin/drelephant/tuning/hbt/SparkConfigurationConstants.java

```diff
@@ -1,4 +1,4 @@
-package com.linkedin.drelephant.tuning.engine;
```

Comment:
Lets not move this to hbt package. Keep it either in util or some other common package. I think most of these constants would be useful for Spark OBT as well